verilog/CST/declaration.cc

```diff
@@ -140,9 +140,15 @@ const verible::SyntaxTreeNode& GetParenGroupFromModuleInstantiation(
 const verible::SyntaxTreeLeaf&
 GetUnqualifiedIdFromVariableDeclarationAssignment(
     const verible::Symbol& variable_declaration_assign) {
-  return *AutoUnwrapIdentifier(*ABSL_DIE_IF_NULL(
-      GetSubtreeAsSymbol(variable_declaration_assign,
-                         NodeEnum::kVariableDeclarationAssignment, 0)));
+  const verible::Symbol* identifier = GetSubtreeAsSymbol(
+      variable_declaration_assign, NodeEnum::kVariableDeclarationAssignment, 0);
+  if (identifier->Kind() == verible::SymbolKind::kLeaf) {
+    // This is a workaround for the below:
+    // TODO(fangism): remove this condition after fixing the issue for "branch".
+    // "riscv_instr          branch;"
+    return verible::SymbolCastToLeaf(*ABSL_DIE_IF_NULL(identifier));
```

Comment:
`ABSL_DIE_IF_NULL(identifier)` is pointless if you already dereference the variable at line 145

Maybe wrap the whole `GetSubtreeasSymbol` in it?